src/index.ts

```diff
@@ -1607,14 +1607,22 @@ function FlatpickrInstance(
                 e.preventDefault();
                 target.focus();
               } else if (childElementCount <= expectedChildElementCount) {
-                self.element.focus();
+                if (self.altInput) {
```

Comment:
We could simply do `self._input.focus()`

Reply:
good call